### ISSUE TITLE
How To `Adding a list partition` ?

### ISSUE THREAD

**Issue by liaozd** [SectorLabs/django-postgres-extra#212]
I am not very clear about the [Using migration operations](https://django-postgres-extra.readthedocs.io/en/latest/table_partitioning.html#using-migration-operations) 

should I edit the migrations files? 

or the [Using the schema editor](https://django-postgres-extra.readthedocs.io/en/latest/table_partitioning.html#using-the-schema-editor), where to  put this code `connection.schema_editor().add_list_partition(` ?

**Comment by Photonios** [comment]
Yes, you have to manually create migration files or write a script that uses the schema editor directly. Your choice.

Table partitioning is an advanced feature. You need to be deeply familiar with both Django and Postgres to safely use it.
